pkg/controller/statefulset/stateful_pod_control_test.go

```diff
@@ -552,30 +553,53 @@ func TestStatefulPodControlUpdatePodClaimForRetentionPolicy(t *testing.T) {
 		})
 		set := newStatefulSet(3)
 		set.GetObjectMeta().SetUID("set-123")
-		pod := newStatefulSetPod(set, 0)
-		claims := getPersistentVolumeClaims(set, pod)
-		for k := range claims {
-			claim := claims[k]
-			indexer.Add(&claim)
+		pod0 := newStatefulSetPod(set, 0)
+		claims0 := getPersistentVolumeClaims(set, pod0)
+		for k := range claims0 {
+			claim := claims0[k]
+			if err := indexer.Add(&claim); err != nil {
+				t.Errorf("Could not add claim %s: %v", k, err)
+			}
+		}
+		pod1 := newStatefulSetPod(set, 1)
```

Comment:
This is basically two tests in one. Can we parametrize this?

Reply:
done